### PR TITLE
feat: make esco identifier input readonly

### PR DESCRIPTION
**Pull request by jona159**

### CODE REVIEW

**Review by YouQam — COMMENTED** [review]
Since `Esco-Identifier` input is readonly, don't you think that it makes sense to hide it for manual added competencies?

File: src/app/components/input.component.ts [hunk 1]
```diff
@@ -39,6 +39,7 @@ import { HlmPDirective } from './spartan/ui-typography-helm/src/lib/hlm-p.direct
 			[attr.maxlength]="maxchar"
 			[attr.max]="max"
 			[type]="fieldType"
+			[readonly]="readonly || false"
```
Comment (YouQam) [inline src/app/components/input.component.ts:42]: Can you elaborate what is the purpose of adding `| | false` ?

Comment (timber-they) [reply src/app/components/input.component.ts:47]: This is a good question!

File: src/app/issuer/components/badgeclass-edit-form/badgeclass-edit-form.component.html [hunk 2]
```diff
@@ -192,6 +192,8 @@ <h2 hlmH2 class="tw-font-bold">2. {{ 'CreateBadge.enterDetails' | translate }}</
 		</div>
 	</div>
 
+	<div class="section tw-text-oebblack tw-px-1" *ngIf="badgeCategory === 'competency'">
```
Comment (timber-they) [inline src/app/issuer/components/badgeclass-edit-form/badgeclass-edit-form.component.html:195]: This seems duplicated?

File: src/app/issuer/components/badgeclass-edit-form/badgeclass-edit-form.component.html [hunk 3]
```diff
@@ -429,9 +431,15 @@ <h2 class="tw-whitespace-pre-wrap tw-text-oebblack tw-font-bold">
 							>
 							</oeb-input>
 
+							<oeb-input
```
Comment (timber-they) [inline src/app/issuer/components/badgeclass-edit-form/badgeclass-edit-form.component.html:434]: This also seems duplicated

File: src/app/issuer/components/badgeclass-edit-form/badgeclass-edit-form.component.ts [hunk 4]
```diff
@@ -43,6 +43,7 @@ export class BadgeClassEditFormComponent extends BaseAuthenticatedRoutableCompon
 	baseUrl: string;
 	badgeCategory: string;
 
+	// Translation
```
Comment (timber-they) [inline src/app/issuer/components/badgeclass-edit-form/badgeclass-edit-form.component.ts:46]: This also seems duplicated

**Review by timber-they — COMMENTED** [review]
Somehow there seem to be quite a few duplicates. What's going on there?

**Comment by jona159** [comment]
Moved to #766

**Pull request closed without merging**